lms/templates/verify_student/make_payment_step.underscore

```diff
@@ -77,6 +78,13 @@
           <% } %>
         <% } %>
       </ul>
+
+      <p><%= gettext("Once payment has been processed, you commit to taking a remotely proctored exam following the protocol defined for this course. Please check that your equipment is compatible with the proctoring system of our ProctorU provider by clicking <a href='http://www.proctoru.com/testitout/' target='_blank'>this link</a>.") %></p>
+      <p>
+        <input class="understand-requirement" type="checkbox" style="display: inline;" /> <%- gettext("I wish to take an exam to validate this course and obtain a certificate") %><br />
+        <input class="understand-requirement" type="checkbox" style="display: inline;" /> <%- gettext("I can schedule the remote exam at the dates defined for this course") %><br />
+        <input class="understand-requirement" type="checkbox" style="display: inline;" /> <%- gettext("My computer equipment is compatible with the exam proctoring tools") %><br />
+      </p>
     </div>
     <% } %>
 
```

Comment:
`<label for="checkbox">` a des règles de CSS non compatibles avec cette implémentation. On a vraiment besoin de ces balises `<label>` ? A quoi servent-elles ?